sdks/python/apache_beam/io/avroio.py

```diff
@@ -60,6 +63,8 @@
 from apache_beam.io.iobase import Read
 from apache_beam.transforms import PTransform
 
+standard_library.install_aliases()
```

Comment:
Is this needed?

Reply:
This is needed for `import io`, but is misplaced.

Reply:
`io` is [available](https://docs.python.org/2/library/io.html) starting from Python 2.6.

Reply:
Removed

Reply:
I removed all the `install_aliases` caused by `io`